lib/src/core/authentication.ts

```diff
@@ -16,23 +16,33 @@
  * under the License.
  */
 
-import { AsgardeoAuthClient, AuthClientConfig, CryptoUtils, Store } from "@asgardeo/auth-js";
-import { AsgardeoAuthException } from "../exception";
-import { AuthURLCallback, NodeTokenResponse } from "../models";
-import { UserSession } from "../session";
+import {
+    AsgardeoAuthClient,
+    AsgardeoAuthException,
+    AuthClientConfig,
+    BasicUserInfo,
+    CryptoUtils,
+    CustomGrantConfig,
+    DecodedIDTokenPayload,
+    FetchResponse,
+    OIDCEndpoints,
+    SessionData,
+    Store,
+    TokenResponse
+} from "@asgardeo/auth-js";
+import { DataLayer } from "@asgardeo/auth-js/dist/src/data";
```

Comment:
```suggestion
import { DataLayer } from "@asgardeo/auth-js";
```

Reply:
Just found out this issue.

> import { DataLayer } from "@asgardeo/auth-js/dist/src/data";

This refers to `src/data/data-layer.ts`

> import { DataLayer } from "@asgardeo/auth-js";

This refers to `src/models/data.ts`
Only `src/data/data-layer.ts` has the interface with `userId` parameter.

Reply:
Fixed with `@asgardeo/auth-js": "^2.0.6`